components/board/genericlinux/gpio.go

```diff
@@ -219,11 +222,29 @@ func (pin *gpioPin) Close() error {
 	return err
 }
 
+// TODO: make sure this matches the other signature
```

Comment:
Implement this! It's important, and Mac users won't be able to compile with out it.